server/routes/updateGoal/updateGoalController.ts

```diff
@@ -60,14 +61,29 @@ export default class UpdateGoalController {
       const nextStepNumber = currentHighestStepNumber + 1
       const newStep: UpdateStepForm = { stepNumber: nextStepNumber, status: 'NOT_STARTED' }
       updateGoalForm.steps.push(newStep)
-      req.session.updateGoalForm = updateGoalForm
       // Redirect back to the Update Goal page with named anchor taking the user straight to the new step
       return res.redirect(`/plan/${prisonNumber}/goals/${goalReference}/update#steps[${nextStepNumber - 1}][title]`)
     }
 
+    return res.redirect(`/plan/${prisonNumber}/goals/${goalReference}/update/review`)
+  }
+
+  getReviewUpdateGoalView: RequestHandler = async (req, res, next): Promise<void> => {
+    const { prisonerSummary } = req.session
+    const { updateGoalForm } = req.session
+
+    const view = new ReviewView(prisonerSummary, updateGoalForm)
+    res.render('pages/goal/update/review', { ...view.renderArgs })
+  }
+
+  submitReviewUpdateGoal: RequestHandler = async (req, res, next): Promise<void> => {
+    const { prisonNumber } = req.params
+    const { updateGoalForm } = req.session
+
     const updateGoalDto = toUpdateGoalDto(updateGoalForm)
     await this.educationAndWorkPlanService.updateGoal(prisonNumber, updateGoalDto, req.user.token)
-    // TODO - RR-188 - handle API error response when updating goal
+
```

Comment:
Can you add the todo comment in please? Otherwise we will forget this is one of the areas that we need to come back to when we do RR-188
```
// TODO - RR-188 - handle API error response when updating goal
```

Reply:
Added the comment back in!